src/test/resources/org/jabref/logic/importer/fetcher/collection_of_computer_science_bibliographies_multiple_results.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<?xml-stylesheet title="XSL_formatting" type="text/xsl" href="http://liinwww.ira.uka.de/bibliography/rss.xsl"?><rss version="2.0">
+
+    <channel xmlns:dc="http://purl.org/dc/elements/1.1/">
+
+        <title>CCSB: +"effective java" +"joshua bloch" +"java series"</title>
+        <link>http://liinwww.ira.uka.de/bibliography/#search</link>
+        <description>Search results in The Collection of Computer Science Bibliographies for query: +"effective java" +"joshua bloch" +"java series"</description>
+        <language>en</language>
```

Comment:
I think, this should be `english` - if possible!

Source: https://tex.stackexchange.com/a/108019/9075

Reply:
This is the exact xml which is returned from doing a query, in this case from viewing the page source on [this page](http://liinwww.ira.uka.de/bibliography/rss?query=%2b%22effective%20java%22%20%2b%22joshua%20bloch%22%20%2b%22java%20series%22&sort=score)